src/material-experimental/mdc-list/list-option.html

```diff
@@ -1 +1,25 @@
-TODO: Implement.
+<!-- Save icons and unclassified content to be places later -->
+<ng-template #icons>
+  <ng-content select="[mat-list-avatar],[matListAvatar],[mat-list-icon],[matListIcon]"></ng-content>
+</ng-template>
+<ng-template #unsortedContent>
+  <ng-content></ng-content>
+</ng-template>
+
+<!-- Prefix -->
+<span class="mdc-list-item__graphic" *ngIf="checkboxPosition !== 'after' else icons">
```

Comment:
Just occurred to me now: what would you think of using the flexbox `order` property to control whether the graphic vs. the checkbox appears at the beginning or the end of the content? The trade-off would be probably adding an extra DOM element as a container, but it would avoid the `ng-template` / `ngIf`

https://jsbin.com/gibogisuci/edit?html,css,output

Reply:
`order` doesn't change the tab order though